src/main/resources/application.yml

```diff
@@ -37,7 +37,7 @@ exceptionmanager:
 
 management:
   endpoints:
-    enabled-by-default: false
+    access: default
```

Comment:
On the previous one this was enabled by default false. Is this the same thing now?